MEMBER_TRAVEL_FUND.md

```diff
@@ -0,0 +1,178 @@
+# OpenJS Foundation Membership Travel Fund
+
+## Purpose
+
+To establish and administer a fund for members of the OpenJS Foundation to travel
+and spread knowledge about and support the Javascript ecosystem and the Foundation.
+
+### [Current Year Fund](https://github.com/openjs-foundation/cross-project-council/blob/master/TRAVEL_FUND/2020.md)
+
+### [2019 funds](https://github.com/openjs-foundation/cross-project-council/blob/master/TRAVEL_FUND/Achived_Travel_Fund_Records/2019.md)
+
+## Requirements
+
+* Candidates must be an active participant of one of the projects of the OpenJS Foundation.
+* Those requesting funds must indicate that they do not have funding available from another
+  source, such as an employer or the event itself that might cover costs for presenters.
+
+## Process
+
+Requests must be made prior to the start of travel. Requests submitted once travel is
+complete will not be approved.
+
+It is strongly recommended that you submit your request far enough in advance to get
+approval before making relevant expenditures. Review of requests will take at least 72
+hours and may take considerably longer. Note, with limited funds available, approval
+is not guaranteed. If a request is rejected, reimbursement will not be issued. The
+Foundation cannot pre-pay for travel expenditures.
```

Comment:
What I would prefer here is that requests made in regard to specific project events are still directed to the steering bodies for each project, rather than to the CPC. The reason is those steering committees will have the most context for determining approval of a request.

For example: If I am attending a Node.js Diagnostic Working Group face-to-face and want to request travel funds, that request should go to the Node.js TSC/CommComm for approval as it is now. If I am attending a Foundation event (such as an OpenJSF Collaborator's Summit) then the request would go to the CPC.

For those who use corporate expense reimbursement type systems, it's no different really than having different approvers for expenses that have different tags.

In advance of a project specific event, the project should submit a request to the CPC for an earmarked allocation for that event. For instance, "Hey, we have a diagnostic working group face to face coming up, we expect about 20 people to attend, please earmark about $5000 for the event to cover travel requests".

Reply:
That makes sense. As I mentioned [above](https://github.com/openjs-foundation/cross-project-council/pull/441#issuecomment-574440617), many of these projects are independently collecting and managing their own funds. They should be tapping into those funds first and not using this travel fund. By contacting their own project committees they can determine if project funds are available before asking for foundation funds.

Reply:
@jasnell this was discussed in the CPC meeting today. One suggestion is that we could land the current PR as is so that we can have the CPC approve travel requests that come in (versus the current state of confusion of how to get approval).  We could then work on an extension that incorporates what you suggested in terms of projects asking for allocations and then managing the approvals themselves.  The projects would then be able to make that ask and decide/implement the approval process in their own repos.